docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -0,0 +1,110 @@
+---
+id: troubleshooting
+title: Troubleshooting
+description: "This page lists common issues with Desktop Modeler and potential resolutions."
+---
+
+## How to start Desktop Modeler without plugins
+
+You can start Desktop Modeler with the [`--disable-plugins` flag](./flags/flags.md#disable-plug-ins).
+
+## How to obtain Desktop Modeler logs
+
+Depending on your operating system, you can find Desktop Modeler logs in different places:
+
+### Windows
+
+```plain
+%APPDATA%\Camunda Modeler\logs
+```
+
+### MacOS
+
+```plain
+~/Library/Logs/Camunda Modeler
+```
+
+### Linux
+
+```plain
+~/.config/camunda-modeler/logs
+```
+
+To produce logging output, you can also run Desktop Modeler from the command line.
+
+## I cannot connect to Zeebe
+
+You try to connect (i.e., to deploy) to a remote Zeebe instance, and Desktop Modeler tells you it "cannot find a running Zeebe."
```

Comment:
Is this exactly the message it gives you?

Reply:
No. But we'll adjust to the actual message as a follow-up.